fairlearn/metrics/_disparities.py

```diff
@@ -7,6 +7,46 @@
 from ._metric_frame import MetricFrame
 
 
+def average_odds_difference(
+        y_true,
+        y_pred,
+        *,
+        sensitive_features,
+        method='between_groups',
+        sample_weight=None) -> float:
+    """Calculate the average odds difference.
+
+    The average of two metrics: `true_positive_rate_difference` and
+    `false_positive_rate_difference`. The former is the difference between the
+    largest and smallest of :math:`P[h(X)=1 | A=a, Y=1]`, across all values :math:`a`
+    of the sensitive feature(s). The latter is defined similarly, but for
+    :math:`P[h(X)=1 | A=a, Y=0]`.
+    The equalized odds difference of 0 means that all groups have the same
```

Comment:
This should probably say "average odds"?

Reply:
Yep